common/src/main/res/layout/dialog_adaptive.xml

```diff
@@ -36,6 +36,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginTop="20dp"
+        android:layout_marginBottom="10dp"
```

Comment:
This adds a bit of space between a title and a button in case there is no message.